eng/pipelines/runtime.yml

```diff
@@ -452,4 +516,31 @@ jobs:
       liveCoreClrBuildConfig: checked
       dependsOnTestBuildConfiguration: ${{ variables.debugOnPrReleaseOnRolling }}
       dependsOnTestArchitecture: x64
-      condition: or(eq(dependencies.checkout.outputs['SetPathVars_coreclr.containsChange'], true), eq(variables['isFullMatrix'], true))
+      condition: >-
+        or(
+          eq(dependencies.checkout.outputs['SetPathVars_coreclr.containsChange'], true),
+          eq(variables['isFullMatrix'], true))
+
+#
+# Libraries Test Execution against a checked runtime
+# Only if CoreCLR or Libraries is changed
```

Comment:
Why not for installer changes?

Reply:
The libraries tests are currently completely unaffected by the installer build. The libraries test host still uses a LKG dotnet.exe, hostfxr, and hostpolicy.

Reply:
Then this probably deserves a TODO :)

Reply:
Whenever we use the installer build to run tests we can run libraries tests with a release coreclr whenever libraries or installer is changed. I wouldn't do it against a checked coreclr.

Reply:
> Then this probably deserves a TODO comment :)

I think it would be better to have an issue tracking that and discuss there. (I think we might already have one, but not sure and can't find it).

Reply:
I think you're looking for [#487 "Use the "live" NETCoreApp shared framework in CoreCLR and Libraries tests"](https://github.com/dotnet/runtime/issues/487).

Reply:
Thanks @dagood -- added a comment there.